README.md

```diff
@@ -170,16 +331,3 @@ $ bundle exec rspec -fd
 Notes:
 + ```-fd``` stands for full details. You can also run tests by using commands ```bundle exec rspec``` or ```bundle exec rake``` to get an overview without details of each test.
 + If you are having trouble running bundle install, please make sure Nokogiri gem is installed on your machine. Full instructions can be found [here](http://www.nokogiri.org/tutorials/installing_nokogiri.html).
-
-### Running the linters
```

Comment:
we would want to keep this section. we shouldn't delete it

Reply:
This has been fixed